modules/xz/5.4.5.bcr.5/source.json

```diff
@@ -0,0 +1,25 @@
+{
+    "url": "https://github.com/tukaani-project/xz/releases/download/v5.4.5/xz-5.4.5.tar.gz",
+    "integrity": "sha256-E1yQuTSu6PvA1Gfeh6Bctw1ifaNqvlGMNXqHNwnlt9Y=",
+    "strip_prefix": "xz-5.4.5",
+    "overlay": {
+        "BUILD.bazel": "sha256-m93hU3hOVXXjx2hddz5sgo2rpcyFu8XtQ6CJ0Hd/BBc=",
+        "MODULE.bazel": "sha256-uT1wNawUyQDf33Yk5Cz8v8NEFaqNPIOm8iWGfEjSja0=",
+        "config.lzma-android.h": "sha256-DZeAf1DFgOMEja/SvAuiSKedWvo1NI30M73irJL07t0=",
```

Comment:
Could you replace the overlaid files that haven't changed since the last version with symlinks? That makes it much easier to review each new version.

Reply:
Done.

And this is the diff for `BUILD.bazel`:
```diff
--- 5.4.5.bcr.4/overlay/BUILD.bazel	2024-10-19 13:49:49.654951535 +0900
+++ 5.4.5.bcr.5/overlay/BUILD.bazel	2024-10-19 13:48:57.194347557 +0900
@@ -41,6 +41,14 @@
     ],
 )
 
+config_setting(
+    name = "wasm32",
+    constraint_values = [
+        "@platforms//os:none",
+        "@platforms//cpu:wasm32",
+    ],
+)
+
 expand_template(
     name = "config_linux_arm64",
     out = "config.lzma-linux-arm64.h",
@@ -76,6 +84,7 @@
         ":linux_x86_64": "config.lzma-linux-x86_64.h",
         ":osx_arm64": "config.lzma-osx-arm64.h",
         ":osx_x86_64": "config.lzma-osx-x86_64.h",
+        ":wasm32": "config.lzma-wasm32.h",
         ("@platforms//os:ios", "@platforms//os:watchos", "@platforms//os:tvos"): "apple_config",
         "@platforms//os:windows": "config.lzma-windows.h",
         "//conditions:default": "config.lzma-linux-x86_64.h",
@@ -83,6 +92,7 @@
     out = "src/liblzma/api/config.h",  # minimize the number of exported include paths
     target_compatible_with = selects.with_or({
         ("@platforms//os:android", "@platforms//os:linux", "@platforms//os:osx", "@platforms//os:ios", "@platforms//os:watchos", "@platforms//os:tvos", "@platforms//os:windows"): [],
+        ":wasm32": [],
         "//conditions:default": ["@platforms//:incompatible"],
     }),
 )
@@ -102,10 +112,15 @@
 
 cc_library(
     name = "lzma",
-    srcs = [
-        "src/common/tuklib_cpucores.c",
-        "src/common/tuklib_physmem.c",
-    ] + glob(
+    srcs = select({
+        "@platforms//os:none": [],
+        "//conditions:default": [
+            "src/common/tuklib_cpucores.c",
+            "src/common/tuklib_physmem.c",
+            "src/liblzma/common/stream_decoder_mt.c",
+            "src/liblzma/common/stream_encoder_mt.c",
+        ],
+    }) + glob(
         [
             "src/**/*.h",
             "src/liblzma/**/*.c",
@@ -113,6 +128,8 @@
         exclude = [
             "src/liblzma/check/crc*_small.c",
             "src/liblzma/**/*_tablegen.c",
+            "src/liblzma/common/stream_decoder_mt.c",
+            "src/liblzma/common/stream_encoder_mt.c",
         ],
     ),
     hdrs = [
@@ -129,6 +146,7 @@
     linkopts = select({
         "@platforms//os:android": [],
         "@platforms//os:windows": [],
+        "@platforms//os:none": [],
         "//conditions:default": ["-lpthread"],
     }),
     linkstatic = select({
@@ -231,10 +249,13 @@
 
 cc_library(
     name = "lzma_src_common",
-    srcs = [
-        "src/common/tuklib_exit.c",
-        "src/common/tuklib_progname.c",
-    ],
+    srcs = select({
+        "@platforms//os:none": [],
+        "//conditions:default": [
+            "src/common/tuklib_exit.c",
+            "src/common/tuklib_progname.c",
+        ],
+    }),
     hdrs = glob(["src/common/*.h"]),
     defines = ["HAVE_CONFIG_H"] + select({
         "@platforms//os:windows": [
```